docassemble/assemblylinewizard/data/questions/assembly_line.yml

```diff
@@ -115,15 +117,18 @@ fields:
       - Form that is not filed in a court: other_form
       - Letter: letter
       - Other: other
+    show if: interview.court_related
   - Allowed courts: interview.allowed_courts
     datatype: checkboxes
-    none of the above: True
+    none of the above: False
```

Comment:
IMO this should stay in. Not sure how other jurisdictions are using the weaver yet, but they should still have the ability to say None of the above courts, even if it is still court related.